lttoolbox/att_compiler.cc

```diff
@@ -52,20 +53,20 @@ AttCompiler::clear()
  *       for conversion?
  */
 void
-AttCompiler::convert_hfst(wstring& symbol)
+AttCompiler::convert_hfst(UString& symbol)
 {
-  if (symbol == L"@0@" || symbol == L"ε")
+  if (symbol == (const UChar*)"@0@" || symbol == (const UChar*)"ε")
```

Comment:
C-style casts are bad, especially for pointers. This won't behave correctly. Make such literals constexpr global UString to compare against.